mne/io/brainvision/brainvision.py

```diff
@@ -4,6 +4,8 @@
 # Authors: Teon Brooks <teon.brooks@gmail.com>
 #          Christian Brodbeck <christianbrodbeck@nyu.edu>
 #          Eric Larson <larson.eric.d@gmail.com>
+#          Jona Sassenhagen <jona.sassenhagen@gmail.com>
```

Comment:
I assume this goes here because I introduced one of the bugs fixed in this PR :D